docs/io.cozy.contacts.md

```diff
@@ -41,19 +41,49 @@ The `io.cozy.contacts` doctype is loosely based on the [vCard RFC](https://tools
   - `label?`: {string} A user-provided localized type of instance
   - `primary?`: {boolean} Indicates a preferred-use instance
 - `relationships`: {object} links between documents
+
   - `groups`: {object} groups the contact belongs to
     - `data`: {array} list of groups
       - `_id`: {string} id of the io.cozy.contacts.groups document
-      - `_type`: {string} doctype "io.cozy.contacts.groups" 
-- `metadata`: {object}
-  - `version`: {integer} Used for migrations. Current version is **1**
+      - `_type`: {string} doctype "io.cozy.contacts.groups"
+  - `accounts`: {object}
+    - `data`: {array} list of accounts
+      - `_id`: {string} id of the io.cozy.contacts.accounts document
+      - `_type`: {string} doctype "io.cozy.contacts.accounts"
+
+- `cozyMetaData`: {object}
+
+  - ...`cozyMetadata`: [Document metadata](./README.md#document-metadata)
+  - `sync`: {object}
 
+    - `ded4265b38c54b0683408c76d9ebd`: {object} id of the sourceContactsAccount
+
+      - `konnector`: {string} example : `"google"`
+      - `lastSync`: {date} (example: `"2018-10-19T10:58:37.025688+02:00"`)
+      - `contactsAccountsId`: {string} id of the io.cozy.contacts.accounts object
+      - `id`: {string} id of the remote object
+      - `remoteRev`: {string} latest rev of the remote object
 
 ## `io.cozy.contacts.groups`
 
 Used to group contacts together. A group can obviously have multiple contacts, but contacts can also be part of multiple groups.
 
 - `name`: {string} is the group's public name
-- `trashed`: {boolean} is `true` if the group is marked for removal and will be deleted soon 
+- `trashed`: {boolean} is `true` if the group is marked for removal and will be deleted soon
 - `metadata`: {object}
-  - `version`: {integer} Used for migrations. Current version is **1**
+  - `version`: {integer} Used for migrations. Current version is **1**
+
+## `io.cozy.contacts.accounts`
+
+Used to know to which accounts (Google, Apple, etc) a contact is related.
```

Comment:
```suggestion
Used to know to which accounts (Google, Apple, etc) a contact is related to.
```

Reply:
"to which ... is related to" ?

Reply:
Yep, le verbe est "related to", pas juste "related". Confirmé avec mon anglophone locale :-)

Reply:
oui mais est-ce que le 1er "to" ne suffit pas ?

Reply:
Ah au temps pour moi, j'avais zappé le second "to" après le which, et je suppose que la personne avec qui j'ai vérifié aussi. Je pense que "Used to know which accounts (Google, Apple, etc) a contact is related to." est plus naturel mais peu importe^^

Reply:
done!